### PR TITLE
Added popular VScode theme Night Owl

### PR DESCRIPTION
**Pull request by SimpleIndian**
I also the formatted readme file.
@RunDevelopment , please review these changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,89 +44,92 @@ Thank you so much for contributing!!
 
 ## Available themes
 
-* [__CB__](themes/prism-cb.css) (originally by [C. Bavota](https://bitbucket.org/cbavota), adapted by [atelierbram](https://github.com/atelierbram))<br />
```
Comment (RunDevelopment) [inline README.md:47]: Please revert all formatting changes to this file.

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Thank you for the PR.

I have one comment and then there is licensing. You are not the owner of this theme. @sdras licensed the theme using MIT, so we have to adhere to that license. Please add a comment to the CSS file crediting @sdras along with [these lines](https://github.com/sdras/night-owl-vscode-theme/blob/main/LICENSE#L1-L3) of the license.

**Comment by souvik362** [comment]
@RunDevelopment , Thanks for the generous suggestion . Quickly fixing it.

**Comment by RunDevelopment** [comment]
I'm really sorry for the delay @SimpleIndian!

LGTM;

**Review by RunDevelopment — APPROVED** [review]

**Pull request merged**